UPGRADE-8.0.md

```diff
@@ -352,6 +352,27 @@ Serializer
  * Remove `AdvancedNameConverterInterface`, use `NameConverterInterface` instead
  * Remove the `CompiledClassMetadataFactory` and `CompiledClassMetadataCacheWarmer` classes
 
+Validator
+---------
+
+ * Remove `Bic::INVALID_BANK_CODE_ERROR` constant
+
+   *Before*
```

Comment:
for other changelog part (your other recent PRs), I totally understand and value such before/after (for readers, for rector, etc etc),

but for this one is it worth it? just a sentence stating to remove the constant without replacement seems enough